app/src/main/res/values/dimens.xml

```diff
@@ -240,6 +240,7 @@
     <dimen name="edit_individual_category_padding_end">0dp</dimen>
     <dimen name="edit_individual_category_padding">10dp</dimen>
     <dimen name="edit_individual_category_margin_top">20dp</dimen>
+    <dimen name="edit_individual_category_menu_margin">8dp</dimen>
```

Comment:
For future reference (this is fine here), you could just put the 8dp in the layout itself. Typically it's nice to have "standard" padding values that are more generic eg. `padding_1` `padding_2` etc etc, that bucket things into the 8, 16, 24 point grid, but this project went overboard with very specific dimens that get declared and then only used like one time.